src/main/java/org/folio/innreach/domain/service/impl/RequestServiceImpl.java

```diff
@@ -264,11 +265,16 @@ private void cancelTransaction(InnReachTransaction transaction) {
   @Override
   public void createRecallRequest(UUID recallUserId, UUID itemId) {
     var pickupServicePoint = getDefaultServicePointIdForPatron(recallUserId);
+    var item = itemService.getItemByHrId(itemId.toString());
+    var holding = item == null ? null : holdingsService.find(item.getHoldingsRecordId()).orElse(null);
```

Comment:
We store identifiers of instance, holding and item in `TransactionHold` when initial request is successfully created. So, at this point there is no need to load the records from inventory to get them, you can relay on these values `folioItemId`, `folioHoldingId`, `folioInstanceId`

Reply:
It is done, thank you!